<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmMain</class>
 <widget class="QWidget" name="FrmMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>283</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>446</width>
    <height>283</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>446</width>
    <height>283</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FrmMain</string>
  </property>
  <widget class="QLineEdit" name="edtBase">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Base Level:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtDifficulty">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Difficulty:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtDurability">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Durability:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtID">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Id:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtReal">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Real Level:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtQuality">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Max Quality:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtCraft">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="placeholderText">
    <string>Suggested Craftsmanship:</string>
   </property>
  </widget>
  <widget class="QListWidget" name="lwOut">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGenerate">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Generate</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
